scripts/jenkins/qat/k8s-install.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+#
+# Copyright 2019 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Installs a k8s cluster (single node) using cloud-native-setup scripts in
```

Comment:
There's no qat specific here so this could be `/common/`. Is it possible to get the systems with kubernetes already set up?

Reply:
We can try moving this installation to the cloud init script and see if we don't reach time out for jenkins.

Reply:
cloud-init script is now in charge to install a k8s cluster when the machine is provisioned. so no need to have anymore the script and stage for that in the jenkinsfile.